docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -1,57 +1,332 @@
+    network. To realize this connection, you need the `id` of said
+    network. Go ahead and list all available networks...
+
+    ```bash
+    openstack network list -f json
+    ```
+
+    ...and look for one called `ext-net`:
+
+    ```plain
+    [
+      {
+        "ID": "0511be8c-eb14-4a69-aa6f-d5886efda140",
+        "Name": "nordostbahnhof",
+        "Subnets": [
+          "9a25e897-6938-4c06-985c-deeb4245c753"
+        ]
+      },
+      {
+        "ID": "201d458b-9b47-4408-9736-980bec77d405",
+        "Name": "nuremberg",
+        "Subnets": [
+          "1b0822b3-62e8-4b40-92e8-8544c72d4c15"
+        ]
+      },
+      {
+        "ID": "2aec7a99-3783-4e2a-bd2b-bbe4fef97d1c",
+        "Name": "ext-net",
+        "Subnets": [
+                ...
+        ]
+      }
+    ]
+    ```
+
+    Look for the `ID` label above the `Name` label of the `ext-net`
+    network, and assign the `ID` value to an environment variable:
+
+    ```bash
+    EXT_NET_ID="2aec7a99-3783-4e2a-bd2b-bbe4fef97d1c"
+    ```
+
+    Now, connect the `nuremberg_router` to the external network like this:
+
+    ```bash
+    openstack router set $ROUTER_ID --external-gateway $EXT_NET_ID
+    ```
+
+    Please note that if the command above is successful, you will get
+    no output on your terminal. There is one last step to take, and
+    that is to connect router `nuremberg_router` to the subnet
+    `nuremberg_subnet` of network `nuremberg`:
+
+    ```bash
+    openstack router add subnet $ROUTER_ID $SUBNET_ID
+    ```
+
+    Again, if the command above is successful, you will get no output.
+
+## Listing networks and getting information
+
+At any time, you may connect to the {{extra.gui}}, list all networks
+you have already created, and get detailed information for any of
+these networks. Alternatively, you may get all that information using
+the OpenStack CLI.
+
+=== "{{extra.gui}}"
+    You may see all defined networks, in all supported regions, by
+    selecting _Networking_ > _Networks_ (see the left-hand side pane
+    on the {{extra.gui}}).
+
+    ![All networks in all regions](assets/new-net/shot-06.png)
+
+    For more information regarding a specific network, click the
+    corresponding three-dot icon (right-hand side) and select _View
+    details_. Then, you can glance over all the details regarding the
+    selected network's ports, subnets, and routers.
+
+    ![Network details](assets/new-net/shot-07.png)
+=== "OpenStack CLI"
+    To list all available networks in a specific region, just type:
+
+    ```bash
+    openstack network list -f json
+    ```
+
+    You can always ask for more specific results. For instance, to see
+    all internal networks only, type the following:
+
+    ```bash
+    openstack network list --internal -f json
+    ```
+
+    You can also get detailed information about a specific network:
+
+    ```bash
+    openstack network list --name nuremberg --long -f json
+    ```
+
+    ```plain
```

Comment:
This JSON output shouldn't be here, but whenever you do want to include JSON output, the proper syntax highlighting hint would be `json`, not `plain`.